src/tracer.ts

```diff
@@ -72,6 +72,7 @@ export class WorkerTracer implements Tracer {
 			},
 			resource: this.resource,
 			spanContext,
+			parentSpanContext,
```

Comment:
This change is because of this: https://github.com/open-telemetry/opentelemetry-js/commit/1afa8ba5140186c8b2338c40e185df6f0b7aff12.